wunderio/next4drupal-project
# Next-13: Add recipes to the project for basic configuration

### Recipe for basic setup and readme

This PR adds: 

* tooling to run recipes (lando command, lando script, patch to composer.json)
* `wunder_next_setup` recipe that will install the needed modules + add pathauto settings for the article content type (basically the steps detailed here: https://next-drupal.org/learn/quick-start
* Initial version of a readme with all the steps to get to the initial working state for both drupal and next
* removal of configuration in config/sync

This PR does not include the setup of languages, paragraphs, media, user permissions, preview  etc.. I suggest adding those one by one at the same time in next and in drupal.


#### how to test

Follow the steps on README.md (in the project root) and verify that you are able to have the basic setup running.
